packages/preview/touying-brandred-uobristol/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "touying-brandred-uobristol"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["HPDell <huyg0180110559@outlook.com>"]
+license = "MIT"
+description = "Touying Slide Theme for University of Bristol"
+homepage = "https://github.com/HPDell/touying-uobristol"
```

Comment:
Just repository suffices if they are the same.
```suggestion
```

Reply:
This issue has been fixed.